src/main/resources/oss/fosslight/repository/OssMapper.xml

```diff
@@ -658,6 +661,14 @@
 			)
 		</if>
 		<if test="@oss.fosslight.util.StringUtil@notEquals(versionCheck,'N')">
+
+
```

Comment:
Please delete empty lines

Reply:
Thank you for your delicacy. I resolve this!